README.md

```diff
@@ -30,14 +30,38 @@ Full documentation [is available here](http://docs.docker.com/swarm/).
 You can download and install from source instead of using the Docker
 image. Ensure you have golang, godep and the git client installed.
 
+Note: The current developing version (0.4) requires golang 1.4.2 version
```

Comment:
I would prefer if we don't mention 0.4 here. it will get out of date pretty quickly otherwise.